index.html

```diff
@@ -50,7 +50,7 @@ <h1 class="my_name">
         I’m Karam<br />
         Glad to see you!
       </h1>
-      <p class="my_details">
+      <p class="my_details my-d">
```

Comment:
- [ ] Your fonts are not displaying properly because they have not been imported yet in the `head` section of your HTML file.   Kindly go [here](https://fonts.google.com/specimen/Poppins) to check out the code that you will need to include the "Poppins" font in your project.   Don't forget to include all the font weights that you will need! 😃 
You could use something like this:
```HTML
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500;700&display=swap" rel="stylesheet"> 
```